docs/maintenance.md

```diff
@@ -113,6 +114,9 @@ The following tools depend on kubernetes, use appropriate version associating to
 - [kind](https://github.com/kubernetes-sigs/kind/releases)
 - [minikube](https://github.com/kubernetes/minikube/releases)
 
+Update `cri-dockerd` to a version later than the one specified in CRI_DOCKERD_VERSION in the minikube's Dockerfile shown below.
+https://github.com/kubernetes/minikube/blob/master/deploy/kicbase/Dockerfile
```

Comment:
IIUC, we shoud allow not only the newer version of cri-docker than the one used by Minikube, but also the same version as the one in Minikube.

```suggestion
Update `cri-dockerd` to a version specified in CRI_DOCKERD_VERSION in [the minikube's Dockerfile](https://github.com/kubernetes/minikube/blob/master/deploy/kicbase/Dockerfile) or newer.
```